### DISCUSSION TITLE
Homework 7

### DISCUSSION THREAD

**Discussion by OlgaKuzminichna** [cafawo/FinancialDataAnalytics#16] [Q&A]
Does anybody faced with this message?
I simply copied an example from deribit and now wonder why method could be not found

![image](https://github.com/cafawo/FinancialDataAnalytics/assets/120052171/daf35ed0-84d7-4878-b694-7efa95e0f53d)

**Comment by onurakinsir** [comment]
You might be misusing or missing parameter(s) or please make sure that you use the correct method/parameters for the specific endpoint of API.

Note that, each endpoint has a different parameter(s) or might be.

**Comment by birte0** [comment]
Hey, in case this issue is still unsolved, please post the code for your request here.
